authority-portal-frontend/src/app/pages/catalog-page/broker-data-offer-cards/broker-data-offer-cards.component.html

```diff
@@ -14,9 +14,38 @@
           {{ dataOffer.organizationName }}
         </span>
       </div>
-      <app-connector-status-led
-        class="mt-1"
-        [status]="dataOffer.connectorOnlineStatus"></app-connector-status-led>
+      <div class="flex items-center gap-1">
+        <app-connector-status-led
+          [status]="dataOffer.connectorOnlineStatus"></app-connector-status-led>
+        <svg
+          *ngIf="dataOffer.assetDataSourceAvailability === 'LIVE'"
+          class="size-5 cursor-pointer"
+          matTooltip="Available Data Offer"
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor">
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M9 12.75 11.25 15 15 9.75M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z" />
+        </svg>
+        <svg
+          *ngIf="dataOffer.assetDataSourceAvailability === 'ON_REQUEST'"
+          class="size-5 cursor-pointer"
+          matTooltip="On Request Data Offer"
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor">
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="m11.25 11.25.041-.02a.75.75 0 0 1 1.063.852l-.708 2.836a.75.75 0 0 0 1.063.853l.041-.021M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Zm-9-3.75h.008v.008H12V8.25Z" />
+        </svg>
```

Comment:
comment: I'm not quite sure if I like the design but I really don't have a better idea...
Maybe we could combine the availability icon with the status LED?
 => But this would lead to design inconsistencies with the rest of the AP

Maybe we could just annotate on request data offers with the icon and then remove the status led on them?
=> I'm not sure if this is the way but my understanding is that this is not a real data offer so the connector status is not relevant for the user in this case. But then again, the offer is still bound to the connector and disappears if the connector goes offline for too long.

I'll leave this for you to decide if you let it as is or if you change something. Just wanted to get those ideas out.

Reply:
> Maybe we could just annotate on request data offers with the icon and then remove the status led on them?

I think this is a good idea for now. For the on request offers the connector status truly does not even matter. The contact info is all that data offer is there for and that is saved in the uiAssetJson anyway. I will go with this for now.

Reply:
I see you left the status LED in the asset detail dialog itself. Is this an oversight or are we leaving it there for on-request offers? I'm fine with both - I mean, the offers are still tied to the connector even if they are not consumed immediately.

Reply:
yes on purpose